src/gc/env/gcenv.ee.h

```diff
@@ -71,6 +71,9 @@ class GCToEEInterface
 
     static void HandleFatalError(unsigned int exitCode);
     static bool ShouldFinalizeObjectForUnload(AppDomain* pDomain, Object* obj);
+    static MethodTable* GetFreeObjectMethodTable();
+    static size_t GetSizeOfArrayBase();
```

Comment:
These constants are used on very perf sensitive paths. I do not think it is ok to virtualize them. They have to be part of the fixed binary contract between VM and GC.